source/includes/usage-examples/retrieve-code-examples.c

```diff
@@ -0,0 +1,113 @@
+#include <bson/bson.h>
+#include <mongoc/mongoc.h>
+#include <stdio.h>
+
+int
+main (int argc, char *argv[])
+{
+    mongoc_client_t *client;
+    mongoc_collection_t *collection;
+    mongoc_cursor_t *results;
+    char *str;
+    bson_error_t error;
+
+    mongoc_init ();
+
+    client =
+        mongoc_client_new ("<connection string URI>");
+    collection = mongoc_client_get_collection (client, "<database name>", "collection name");
+
+    // start-find
+    {
+        bson_t *query = bson_new ();
```

Comment:
```suggestion
    
    {
        // start-find
        bson_t *query = bson_new ();
```

Suggest putting all `start/end` comments inside the block scopes. Including the `{` and `}` in the rendered examples seems unneeded.